test/media/ttml_text_parser_unit.js

```diff
@@ -395,6 +395,15 @@ describe('TtmlTextParser', function() {
         'end="01:02:03.200">Line1<br/>Line2</p></body></tt>');
   });
 
+  it('inserts a newline on br in elements nested in cue', function() {
```

Comment:
Let's merge this test and a previous one into a single "it('replaces <br> tags with newline characters')" with 2 calls to verifyHelper in it.
(Like in 'disregards empty divs and ps' above which has 3 calls).